test/plugins/validation/oas3/operations.js

```diff
@@ -191,6 +250,16 @@ describe('validation plugin - semantic - operations - oas3', function() {
   });
 
   it('should not crash in request body name check when path name contains a period', function() {
+    const config = {
+      operations: {
+        no_request_body_content: 'error',
+        no_request_body_name: 'warning'
+      },
+      schemas: {
+        binary_string: 'warning'
+      }
+    };
+
```

Comment:
It seems odd that we would have to add this config for existing tests.  Why is this?

Reply:
Since this binary string rule spans multiple semantic validators, we have to get the rule from the shared schema rules with `const configSchemas = config.schemas`.

The parameters, responses, and operations validators previously only used the OAS3-specific rules, but the way it is implemented now requires the shared schema rules to be present. Later it is used:

`const binaryStringStatus = configSchemas.binary_string;
  if (binaryStringStatus !== 'off') {`

I could wrap this with:

`if (configSchemas) {
    const binaryStringStatus = configSchemas.binary_string;`

Reply:
I now see that this answers my comment above. I think we might want to re-consider the approach here. Possibly define separate rules for each section to give more fine-grained control. We can discuss this with @mkistler

Reply:
Added a 'schemas' section to the OAS3 rules and put the json_or_param_binary_string config option there.

Then removed the individual configuration objects in each test and replaced it by importing the default OAS3 rules.